students/17th/hkk/chaehyun-kim/user/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db      import models
+
+# Create your models here.
+
+class User(models.Model):
+   email        = models.EmailField(max_length=100)
+   password     = models.CharField(max_length=20)
+   name         = models.CharField(max_length=10)
+   phone_number = models.CharField(max_length=20)
```

Comment:
```suggestion
   email        = models.EmailField(max_length=100, unique=True)
   password     = models.CharField(max_length=20)
   name         = models.CharField(max_length=10, unique=True)
   phone_number = models.CharField(max_length=20, unique=True)
```
이름이 사용자 명을 의미하는 것이 아닌 ID 라면, 모두 유일값만 유지되도록 해주셔야 합니다.

Reply:
name은 계정을 만든 사용자의 실명을 입력시키려는 의도로 작성했습니다! 필요없는 field 였을려나요...ㅎㅎ unique 지정해서 수정하겠습니다! 그런데 password는 지정하지 않아도 되는건가요??

Reply:
넵! 개념 이해하고 추가하겠습니다!